package.json

```diff
@@ -89,6 +90,7 @@
     "chai": "^4.1.2",
     "codecov": "^3.6.5",
     "deepmerge": "^4.2.2",
+    "es5-validator": "1.3.1",
```

Comment:
m: Could we use https://github.com/yowainwright/es-check instead? It has way more downloads and seems to be more flexible in case we need to expand this in the future.

Reply:
good idea, I replaced it!